src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,40 @@
+import { ChangeEvent } from "react";
+import "./style.scss";
+
+interface FilterProps {
+  genre: String;
+  genreList: Array<object>;
+  chengeGenre(e: React.MouseEvent): void;
+  chengeSort(e: React.ChangeEvent<HTMLSelectElement>): void;
+}
+
+const Filter: React.FC<FilterProps> = ({
+  genre,
+  genreList,
+  chengeGenre,
+  chengeSort,
+}) => {
+  const genreListDom = genreList.map((el: any, index: number) => {
+    return (
+      <li id={el.id} className={genre === el.id ? "active" : ""} key={el.id}>
+        {el.genre}
+      </li>
+    );
+  });
+  return (
+    <div className="filter">
+      <ul className="filter-genre" onClick={chengeGenre}>
```

Comment:
what will happen if user will click on `ul`, but not on some specific li?
i think it is better to put it not specific `li`